src/ontology/mondo-edit.obo

```diff
@@ -420511,24 +420513,21 @@ is_a: MONDO:0005046 {source="Orphanet:310050"} ! immune system disorder
 
 [Term]
 id: MONDO:0017770
-name: Robinow-like syndrome
-def: "Robinow-like syndrome is characterized by the association of the clinical features present in Robinow syndrome (short stature, mesomelic brachymelia, macrocephaly, and hypoplastic genitalia), with anterior chamber cleavage anomalies. It has been described in two sisters and is transmitted as an autosomal recessive trait." [Orphanet:3105]
+name: obsolete Robinow-like syndrome
+def: "OBSOLETE. Robinow-like syndrome is characterized by the association of the clinical features present in Robinow syndrome (short stature, mesomelic brachymelia, macrocephaly, and hypoplastic genitalia), with anterior chamber cleavage anomalies. It has been described in two sisters and is transmitted as an autosomal recessive trait." [Orphanet:3105]
 comment: Reason of obsoletion: out of scope - MONDO:excludeHistoricalDisease. Term to consider: -
```

Comment:
you can remove this comment. Or update it accordingly (eg if there is a term to consider)